spec/webview-spec.js

```diff
@@ -243,6 +243,7 @@ describe('<webview> tag', function () {
     it('preload script can require modules that still use "process" and "Buffer" when nodeintegration is off', async () => {
       const message = await startLoadingWebViewAndWaitForMessage(webview, {
         preload: `${fixtures}/module/preload-node-off-wrapper.js`,
+        nodeIntegration: true,
```

Comment:
why not `sandbox: false`?

Reply:
🤷🏻 sure, seems about the same to me but i'll switch it if you like.